### ISSUE TITLE
Modify template : Modernization of the implementation of ActionMethod of MVC.

### ISSUE THREAD

**Issue by daisukenishino** [OpenTouryoProject/OpenTouryo#215]
# Requirement
https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Controllers/CrudMuController.cs

## Use model binding and validation.
```CS
public ActionResult XXXX(XXXXViewModel model)
{
    // XXXXViewModelの検証
    if (ModelState.IsValid)
    {
```

## Grant following attributes appropriately.
- Basically there is no case corresponding to both GET / POST
  - [HttpGet]  
Initial display uses GET.
  - [HttpPost]  
After that, basically use model binding and use POST.
  - [ValidateAntiForgeryToken]
    - Grant to all POST.
    - And ,add "@Html.AntiForgeryToken()" to the form in the View.
- It also works with Forms authentication.
  - [Authorize]
  - [AllowAnonymous]

### Use of asynchronous controller
Basically it is better to use asynchronous controllers for IO binds.

```CS
public Task<ActionResult> Index()
{
    TestReturnValue ret = await DoBusinessLogicAsync(...);
    return View();
}
```

For this, [there is the needs to asynchronize the invocation method of layer B of OpenTouryo](https://github.com/OpenTouryoProject/OpenTouryo/issues/216).
